pkg/platform/openstack/kuryr_bootstrap.go

```diff
@@ -575,8 +575,12 @@ func BootstrapKuryr(conf *operv1.NetworkSpec, kubeClient client.Client) (*bootst
 	// the API. With healthchecks enabled for the pool we'll get masters added automatically
 	// when they're up and ready.
 	log.Print("Creating OpenShift API loadbalancer pool members")
-	r, _ := regexp.Compile(fmt.Sprintf("^%s-(master-port-[0-9]+|bootstrap-port)$", clusterID))
+	r, _ := regexp.Compile(fmt.Sprintf("^%s-(master-port-[0-9]+|bootstrap-port+|master-[0-9])$", clusterID))
```

Comment:
This expression looks incorrect. I believe it should instead read: `^%s-(master-port-[0-9]+|bootstrap-port|master-[0-9]+)$`. As it's written, "bootstrap-portttttttttttttttttt" will match and "master-10" will not (though "master-port-10" will).

Reply:
Thanks again for spotting this issue. Withh CI cooperation the fix on main branch can be merged by tomorrow and be included in this backport.